chromadb/db/duckdb.py

```diff
@@ -370,9 +370,6 @@ def get_by_ids(self, ids: List, columns: Optional[List] = None):
 
         return response
 
-    def raw_sql(self, sql):
```

Comment:
technically more than just types... but we had talked about removing this, so drive-by. Can revert